pyaedt/icepak.py

```diff
@@ -3789,28 +3787,40 @@ def assign_solid_block(
         object_name : str
             Name of the object.
         power_assignment : str or dict
-            String with value and units of the power assignment or with ``"Joule Heating"``.
-            Also, a dictionary can be used for temperature dependent or transient assignment.
-            The dictionary should contain three keys: ``"Type"``, ``"Function"`` and
-            ``"Values"``. Accepted ``"Type"`` values are: ``"Temp Dep"`` and ``"Transient"``.
-            Accepted ``"Function"`` are: ``"Linear"``, ``"Power Law"``, ``"Exponential"``,
-            ``"Sinusoidal"``, ``"Square Wave"`` and ``"Piecewise Linear"``. ``"Temp Dep"`` only
-            support the latter. ``"Values"`` contains a list of strings containing the parameters
-            required by the ``"Function"`` selection (e.g. ``"Linear"`` requires two parameters:
-            the value of the variable at t=0 and the slope of the line). The parameters required by
-            each ``Function`` option is in Icepak documentation. The parameters must contain the
-            units where needed.
+            String with the value and units of the power assignment or with
+            ``"Joule Heating"``. For a temperature-dependent or transient
+            assignment, a dictionary can be used. The dictionary should contain three keys:
+            ``"Type"``, ``"Function"``, and ``"Values"``.
+            - For the ``"Type"`` key, accepted values are ``"Temp Dep"`` and ``"Transient"``.
```

Comment:
For a list to render properly, you must have a blank line before the first list element and after the last list element if there is more content.